packages/cactus-test-plugin-htlc-eth-besu/src/test/typescript/integration/plugin-htlc-eth-besu/refund-endpoint-invalid.test.ts

```diff
@@ -210,17 +210,17 @@ describe(testCase, () => {
     });
     expect(responseTxId.callOutput).toBeTruthy();
     const id = responseTxId.callOutput as string;
+
     try {
       const refundRequest: RefundReq = {
         id,
         web3SigningCredential,
         connectorId,
         keychainId,
       };
-      const refundResponse = await api.refundV1(refundRequest);
-      expect(refundResponse.status).toEqual(200);
-    } catch (error: any) {
-      expect(error.response.status).toEqual(500);
+      await api.refundV1(refundRequest);
+    } catch (error: unknown) {
+      expect(error).toReject;
```

Comment:
@Leeyoungone I'd refactor this the same way as the one above with the `.rejects.toHaveProperty(["response", "status"], 500);` way.

Reply:
Sorry, I think I altered an extra file on accident here. Let me fix this real quick.